prep-html-syntax/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <title>HTML Syntax Exercise</title>
+
```

Comment:
Be sure to close your `</head>` element before opening your `<body>` element.

Reply:
🤦‍♀️
Updated
![Screenshot HTML syntax (Added Closing -head- Tag)](https://user-images.githubusercontent.com/97073517/148717255-feeb9ee7-e5fd-48fb-ae11-d5b13111d88d.png)
d